docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -136,14 +136,16 @@ Use the Grafana Alerting - Grafana OnCall integration to effortlessly connect al
 
 To learn more, refer to the [Grafana OnCall integration for Alerting documentation](/docs/grafana/next/alerting/alerting-rules/manage-contact-points/configure-oncall/).
 
-## New browse dashboards view
+## New Browse Dashboards
```

Comment:
I'm assuming that we're talking about browsing dashboards in general, so this should be lower case
```suggestion
## New browse dashboards
```